### ISSUE TITLE
Clear screen is always on

### ISSUE THREAD

**Issue by MehdiSaffar** [IngoMeyer441/simple-term-menu#12]
I am using TerminalMenu with default settings, just passing a title and options, but everytime a menu is created it the previous one is cleared. 

I would like the menus to appear below each other is there a way to do that?

Thanks

**Comment by Seluj78** [comment]
Could you provide your code as an example ?

**Comment by Seluj78** [comment]
Also I think you are looking for the inverse of `  -l, --clear-screen    clear the screen before the menu is shown`

**Comment by IngoMeyer441** [comment]
The current `develop` branch adds a new `clear_menu_on_exit` constructor parameter which defaults to True. Could you install the development version with
```bash
python3 -m pip install git+https://github.com/IngoMeyer441/simple-term-menu@develop
```
and set `clear_menu_on_exit` to `False`?

**Comment by IngoMeyer441** [comment]
The `clear_menu_on_exit` is now released (v0.10.5). Please feel free to reopen if it does not work as expected.
